connmgr.go

```diff
@@ -305,6 +355,68 @@ func (cm *BasicConnMgr) trim() {
 	cm.lastTrimMu.Unlock()
 }
 
+func (cm *BasicConnMgr) getConnsToCloseEmergency(target int) []network.Conn {
```

Comment:
Can we somehow prioritize trimming inbound connections?

Reply:
How exactly do you want to do that? First close (unprotected) inbound connections, then outbound connections? Or should we include the tag values in that calculation somehow?

Reply:
I'd sort by tag first, then inbound v outbound. That's not perfect, but there are at least limited ways in which inbound connections can obtain "tags".

Reply:
That's now implemented in the latest iteration :)